src/stdlib/builtins.md

```diff
@@ -147,77 +147,182 @@ The following generic built-ins compile to WebAssembly instructions directly.
+  | f64                                        | f64.min
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *emulated*
+  </details>
 
 * ```ts
   function ceil<T>(value: T): T
   ```
-  Performs the ceiling operation on a 32-bit or 64-bit float.
+  <details><summary>Performs the ceiling operation on a 32-bit or 64-bit float.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.ceil
+  | f64                                        | f64.ceil
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
+  </details>
 
 * ```ts
   function floor<T>(value: T): T
   ```
-  Performs the floor operation on a 32-bit or 64-bit float.
+  <details><summary>Performs the floor operation on a 32-bit or 64-bit float.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.floor
+  | f64                                        | f64.floor
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
+  </details>
 
 * ```ts
   function copysign<T>(x: T , y: T): T
   ```
-  Composes a 32-bit or 64-bit float from the magnitude of `x` and the sign of `y`.
+  <details><summary>Composes a 32-bit or 64-bit float from the magnitude of <code>x</code> and the sign of <code>y</code>.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | f32 | f32.copysign
+  | f64 | f64.copysign
+  </details>
 
 * ```ts
   function nearest<T>(value: T): T
   ```
-  Rounds to the nearest integer tied to even of a 32-bit or 64-bit float.
+  <details><summary>Rounds to the nearest integer tied to even of a 32-bit or 64-bit float.</summary>
+
+  | T                                          | Instruction
+  |--------------------------------------------|-------------
+  | f32                                        | f32.nearest
+  | f64                                        | f64.nearest
+  | i8, u8, i16, u16, i32, u32, i64, u64, bool | *omitted*
+  </details>
 
 * ```ts
-  function reinterpret<T>(value: auto): T
+  function reinterpret<TTo>(value: auto): T
   ```
-  Reinterprets the bits of the specified value as type `T`. Valid reinterpretations are u32/i32 to/from f32 and u64/i64 to/from f64.
+  <details><summary>Reinterprets the bits of the specified value as type <code>T</code>.</summary>
+
+  | TTo      | Instruction
+  |----------|-------------
+  | i32, u32 | i32.reinterpret_f32
+  | i64, u64 | i64.reinterpret_f64
+  | f32      | f32.reinterpret_i32
+  | f64      | f64.reinterpret_i64
+  </details>
 
 * ```ts
   function sqrt<T>(value: T): T
   ```
-  Calculates the square root of a 32-bit or 64-bit float.
+  <details><summary>Calculates the square root of a 32-bit or 64-bit float.</summary>
+
+  | T   | Instruction
+  |-----|-------------
+  | f32 | f32.sqrt
+  | f64 | f64.sqrt
```

Comment:
the same

Reply:
Similar to the one above, in that we do not polyfill an integer version here.